sample-kotlin/src/main/java/com/infullmobile/android/infullmvp/sample/sample_mvp_custom_view/SampleMvpCustomViewView.kt

```diff
@@ -0,0 +1,20 @@
+package com.infullmobile.android.infullmvp.sample.sample_mvp_custom_view
+
+import android.widget.TextView
+import com.infullmobile.android.infullmvp.PresentedCustomView
+import com.infullmobile.android.infullmvp.sample.R
+
+open class SampleMvpCustomViewView(
```

Comment:
Any ideas how to call it?

Reply:
Instead of SampleMvpCustomView call the whole package something else. Maybe SampleMvpCard, or SampleMvpTemperatureCard, SampleMvpPage